### PR TITLE
[dep]Adding libao-dev

### PR DESCRIPTION
**Pull request by KanayaNakamura**
Please add the following dependency to the rosdep database.

## Package name:

libao-dev

## Package Upstream Source:

https://www.xiph.org/ao/

## Purpose of using this:

Development library to implement simple audio output.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/search?keywords=libao-dev
- Ubuntu: https://packages.ubuntu.com/
   - https://packages.ubuntu.com/search?keywords=libao-dev
- Fedora: https://packages.fedoraproject.org/
  - https://packages.fedoraproject.org/pkgs/libao/libao-devel
- Alpine: https://pkgs.alpinelinux.org/packages
  - https://pkgs.alpinelinux.org/packages?name=libao-dev&branch=edge&repo=&arch=&maintainer=

### CODE REVIEW

File: rosdep/base.yaml [hunk 1]
```diff
@@ -2262,6 +2262,11 @@ libann-dev:
   debian: [libann-dev]
   fedora: [ann-devel]
   ubuntu: [libann-dev]
+libao-dev:
+  alpine: [libao-dev]
+  debian: [libao-dev]
+  fedora: [libao-devel]
```
Comment (tfoote) [inline rosdep/base.yaml:2268]:
```suggestion
  fedora: [libao-devel]
  opensuse: [libao-devel]
  rhel: [libao-devel]
```

Add suggested targest from linter

**Review by tfoote — CHANGES_REQUESTED** [review]
This looks good. Please add the two suggestions from the linter. Otherwise it's ready to go.

File: rosdep/base.yaml (hunk 1 above)
Comment (KanayaNakamura) [reply rosdep/base.yaml:2268]: That's a great suggestions. Thanks!

**Review by nuclearsandwich — APPROVED** [review]

**Review by tfoote — APPROVED** [review]
Thanks for the updates

**Pull request merged**